Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/pageObjects/AddressEditAddPage.java

```diff
@@ -0,0 +1,93 @@
+package com.griddynamics.qa.vikta.uitesting.sample.pageObjects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddressEditAddPage {
+
+    @FindBy(id = "aBack")
+    public WebElement aBack;
```

Comment:
if all these WebElements are accessed only in this class, please make them private